CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.3.6 | [PR#3733](https://github.com/bbc/psammead/pull/3733) Minor & patch package updates |
```

Comment:
Just a heads up – because Psammead is a [monorepo](https://en.wikipedia.org/wiki/Monorepo) and changes in this PR relate to one of its packages only (and not Psammead itself), we don't have to bump Psammead and give it a changelog entry. I'd be happy to explain this further if you have any questions or would like to know more.